react/hooks/useShippingOptions.ts

```diff
@@ -97,7 +97,8 @@ const useShippingOptions = () => {
   const onSelectPickup = async (pickup: Pickup) => {
     setSelectecPickup(pickup)
 
-    await updateSession(selectedZipCode!, geoCoordinates!, pickup)
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Is this really necessary?

Reply:
In this case yes, I was double checking the main reason is that if for some reason the pickup changes it will update the pickup info inside of the facets value and with that update the session token as well and this will be used later on on the IS API.

Reply:
I still don't understand the "!" at the end. Is this checking if the field is present? If so, it would be better to wrap this code on a if statement rather than...